packages/README.md

```diff
@@ -1,3 +0,0 @@
-# Components
```

Comment:
This has been deleted in the content has been moved inline to the page (pages/components/index.tsx)